src/gsi/berror.f90

```diff
@@ -615,6 +615,7 @@ subroutine pcinfo
                 if (aircraft_t_bc_pof) obs_count = ostats_t(j,i)
                 if (aircraft_t_bc) obs_count = ostats_t(1,i)
 
+!               if (obs_count>zero) vprecond(nclen1+ii)=vprecond(nclen1+ii)/(one+rstats_t(j,i)*varprd(jj))
```

Comment:
Line 618 is commented out.  May we remove this line?

Reply:
Same as previous comment.  This is a change that I think should actually be made. It slightly improves convergence (at least in the first few iterations). I put it in commented out because it does change the iteration results.

Reply:
Same reply as previous comment.